tasks/models.py

```diff
@@ -18,19 +17,22 @@ class Task(models.Model):
     end_time = models.DateTimeField(null=True)
     deleted_at = models.DateTimeField(null=True)
     level = models.CharField(max_length=1, default="")
+    situations_m2m = models.ManyToManyField("Situation", through="TakeTask")
+    member_id = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, default=1)
 
     objects = TaskManager()
 
     def delete(self):
         self.deleted_at = timezone.now()
         self.save()
 
-
-class TakeTask(models.Model):
-    pass
-
-
 class Situation(models.Model):
     col_name = models.CharField(max_length=20, default="")
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    tasks_m2m = models.ManyToManyField("Task", through="TakeTask")
+
+class TakeTask(models.Model):
```

Comment:
```suggestion
class TakeTask(models.Model):
```
光看這個名稱有點不太知道是什麼，看要不要改成 `task_member` 或 `member_task`